docs/requirements.txt

```diff
@@ -1,3 +1,7 @@
 sphinx==1.7.3
+sphinx-gallery
```

Comment:
[Torchaudio](https://github.com/pytorch/text/blob/master/docs/requirements.txt) and [Torchtext](https://github.com/pytorch/audio/blob/master/docs/requirements.txt) both use sphinx v2.4.4. It might be possible to bump version to sphinx 2.4.4